sdk/wallet-adapter/wallet-standard/src/utils.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { IdentifierString } from "@wallet-standard/core";
+
+export interface TransactionParameters {
+  account: string;
```

Comment:
Let's use `WalletAccount` at the standard level, we can make it strings in the adapter layer.